bundles/org.openhab.binding.cbus/pom.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Original is not being maintained and didnt even build. There are several changes.
1) added interfaces so we could reliably check when the interface is up to fix startup issues
2)  Update the logging and and add a threading factory as per comments from wborn Dec 2017 in  #2935
3) Added new methods to support CBus functionality that wasnt implemented in the original library.
4) Made it build since the dependencies that were referenced by the latest version arent available anymore and in Nov 2017 the maintainer said he might update it but wasnt sure when and there have been no updates since.